routers/user/profile.go

```diff
@@ -192,7 +199,7 @@ func Profile(ctx *context.Context) {
 		ctx.Data["Page"] = paginater.New(int(count), setting.UI.User.RepoPagingNum, page, 5)
 		ctx.Data["Total"] = count
 	default:
-		if len(keyword) == 0 {
+		if false && len(keyword) == 0 {
```

Comment:
Same here

Reply:
It's an intentional and temporary way to "comment out" the if-part making sure the else-part below (with SearchRepositoryByName) gets used. Thought I'd hear some opinions on [this](#issuecomment-480642179) before just cutting the if-part.